pkg/engine/engine.go

```diff
@@ -256,5 +256,9 @@ func fragmentFirstLine(chunk *sources.Chunk) (int64, *int64) {
 	case *source_metadatapb.MetaData_Gerrit:
 		fragmentStart = &metadata.Gerrit.Line
 	}
```

Comment:
**Nit:** A `default` case is a bit more readable to me

```go
// ...
default:
    return 0, nil
}
return *fragmentStart, fragmentStart
```

Reply:
That is much better! Switched it out.

Reply:
That is much better! Switched it out.